params.go

```diff
@@ -172,6 +172,11 @@ type ListParamsContainer interface {
 	GetListParams() *ListParams
 }
 
+type APIMode string
+
+var V1APIMode APIMode = "v1"
```

Comment:
changed from

```go
var PreviewAPIMode APIMode = "preview"
var StandardAPIMode APIMode = "standard"
```

Reply:
The readme is still referring to` stripe.PreviewAPIMode` though